### PR TITLE
Update load-balancing.md

### PR DESCRIPTION
**Pull request by AspirinSJL**

### CODE REVIEW

File: doc/load-balancing.md [hunk 1]
```diff
@@ -129,10 +129,9 @@ works:
         by the resolver. It asks the balancer for the server addresses to
         use for the server name originally requested by the client (i.e.,
         the same one originally passed to the name resolver).
-        - Note: The `grpclb` policy currently ignores any non-balancer
-          addresses returned by the resolver. However, in the future, it
-          may be changed to use these addresses as a fallback in case no
-          balancers can be contacted.
+        - Note: In `grpclb` policy, the non-balancer addresses returned by
```
Comment (markdroth) [inline doc/load-balancing.md]: s/In/In the/

File: doc/load-balancing.md [hunk 2]
```diff
@@ -129,10 +129,9 @@ works:
...
+          the resolver are used as a fallback in case no balancers can be
+          contacted.
```
Comment (markdroth) [inline doc/load-balancing.md]: Please add "when the LB policy is started".

File: doc/load-balancing.md (hunk 1 above)
Comment (AspirinSJL) [reply doc/load-balancing.md]: Done.

File: doc/load-balancing.md (hunk 2 above)
Comment (AspirinSJL) [reply doc/load-balancing.md]: Done.

**Pull request merged**